packages/rulesets/src/spectral/functions/utils.ts

```diff
@@ -90,7 +90,7 @@ export function getProperty(schema: any, propName: string): any {
 }
 
 export function findBodyParam(params: any) {
-  const isBody = (elem: any) => elem.name === "body" && elem.in === "body"
+  const isBody = (elem: any) => elem.name === "body"
```

Comment:
What is this change for? is it auto generated?

Reply:
Nope, it isnt mandatory for the body param's nam eto be "body", it is wrong and fixing it in this PR

Reply:
Oh, in that case shouldn't your change be:

```suggestion
  const isBody = (elem: any) => elem.in === "body"
```
since we don't care about the param name?

Reply:
yup, just fixed it!
thanks